package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sgn-v2-contracts",
-  "version": "1.0.0",
+  "version": "0.1.0",
```

Comment:
why change this?

Reply:
as per discussion with michael, this is for releasing the contracts as an npm package so third-party developers can use install the dependent contracts through `npm install sgn-v2-contracts@0.1.0`. 0.1.0 indicates currently the contract is not stable enough yet